### ISSUE TITLE
Object of class craft\fields\data\SingleOptionFieldData could not be converted to int

### ISSUE THREAD

**Issue by bartrylant** [verbb/formie#1487]
### Describe the bug

When watching certain submissions in the control panel, I get this error:

```
Object of class craft\fields\data\SingleOptionFieldData could not be converted to int

{% tag 'div' with containerAttributes %}
    {%- for key, option in options %}
        {%- if option is not iterable %}
            {%- set option = {label: option, value: key} %}
        {%- endif %}
        <div>
            {% include "_includes/forms/radio" with {
                describedBy: describedBy ?? null,
                name: name ?? null,
 >>>>>               checked: (option.value is defined and option.value == value),
                autofocus: loop.first and (autofocus ?? false) and not craft.app.request.isMobileBrowser(true),
            }|merge(option) only %}
        </div>
    {% endfor %}
{% endtag %}
```

### Steps to reproduce

1. Watch a submission in the control panel.

### Form settings

- Multi-page form: No
- Submission Method: Page Reload
- Client-side Validation: Yes
- Custom Form Templates: No

### Craft CMS version

4.4.14

### Plugin version

2.0.32

### Multi-site?

Yes

**Comment by engram-design** [comment]
I can't seem to replicate this, sorry. Can you let me know your Radio Buttons field settings, and potentially the value for that field in the submission? Does this also happen for every submission, or just some?

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, run `composer require verbb/formie:"dev-craft-4 as 2.0.32"`.

**Comment by engram-design** [comment]
Fixed in [2.0.33](https://github.com/verbb/formie/releases/tag/2.0.33)
